test/suite/run.py

```diff
@@ -415,6 +418,17 @@ def error(exitval, prefix, msg):
                 configfile = args.pop(0)
                 configwrite = True
                 continue
+            if option == '-randomseed' or option == 'R':
+                seedw = random.randint(1, 0xffffffff)
+                seedz = random.randint(1, 0xffffffff)
+                continue
+            if option == '-seed' or option == 'S':
+                if seedw != 0 or seedz != 0 or len(args) < 2:
+                    usage()
+                    sys.exit(2)
+                seedw = args.pop(0)
+                seedz = args.pop(0)
```

Comment:
Should check to make sure that neither seedw and seedz are not 0.  It shouldn't be possible, and will probably cause parts of the code to break.

Reply:
done!